setup.py

```diff
@@ -4,39 +4,36 @@
 from taxi import __version__
 
 install_requires = [
-    'click>=3.3',
-    'appdirs>=1.4.0',
+    "click>=3.3",
```

Comment:
Please don’t change the formatting in this PR. I would be very happy to blacken the code, but this should be done in a separate PR so we can add the commit to ignore revs.

Reply:
Yes sorry, my LSP formats it on its own and sometimes I miss it I'll fix this.